src/lib.rs

```diff
@@ -1,7 +1,7 @@
 //! # nom, eating data byte by byte
 //!
 //! nom is a parser combinator library with a focus on safe parsing,
-//! streaming patterns, and as much as possible zero copy.
+//! streaming patterns, and as minimal copying as possible.
```

Comment:
"zero copy" is actually an important word here, so it should not be removed

Reply:
Reverted